iosHyperskillApp/iosHyperskillApp/Sources/Modules/StepQuiz/ViewData/StepQuizViewDataMapper.swift

```diff
@@ -91,4 +90,35 @@ final class StepQuizViewDataMapper {
             stepHasHints: stepHasHints
         )
     }
+
+    private func resolveQuizType(
+        step: Step,
+        state: StepQuizFeatureStepQuizStateKs,
+        attemptLoadedState: StepQuizFeatureStepQuizStateAttemptLoaded?
+    ) -> StepQuizChildQuizType {
+        let unsupportedChildQuizType = StepQuizChildQuizType.unsupported(blockName: step.block.name)
+
+        if state == .unsupported {
+            return unsupportedChildQuizType
+        }
+
+        let childQuizType = StepQuizChildQuizType(step: step)
+
+        if case .fillBlanks = childQuizType {
```

Comment:
I think that on platform code we need to differ fill blanks with selects and inputs to render different views for these types. It isn't problem now, but we need to solve this in implementation of fill blanks with selects